modules/dynamic-plugins/proc-installing-and-configuring-redis-cache.adoc

```diff
@@ -0,0 +1,37 @@
+[id="proc-installing-and-configuring-redis-cache_{context}"]
+= Using Redis Cache with dynamic plugins
+You can use the Redis cache store to improve {product-very-short} performance and reliability. Plugins in {product-very-short} receive dedicated cache connections, which are powered by Keyv.
+
+== Installing Redis Cache in {product}
+
+.Prerequisites
+* You have installed Red Hat Developer Hub by using either the Operator or Helm chart.
+* You have an active Redis server. For more information on setting up an external Redis server, see the  link:https://www.redis.io/docs/latest/[`Redis official documentation`].
+
+.Procedure
+Add the following code to your `app-config.yaml` file:
+[source, yaml]
+----
+backend:
+  cache:
+    store: redis
+    connection: redis://user:pass@cache.example.com:6379
+    useRedisSets: true
+----
+
+== Configuring Redis Cache in {product}
+=== useRedisSets
+The `useRedisSets` option lets you decide whether to use Redis sets for key management. By default, this option is set to `true`.
+
+When `useRedisSets` is enabled (`true`):
+
+* A namespace for the Redis sets is created, and all generated keys are added to that namespace, enabling group management of the keys.
+
+* When a key is deleted, it's removed from the main storage and the Redis set.
+
+* When using the clear function to delete all keys, every key in the Redis set is checked for deletion, and the set itself is also removed.
+
+[NOTE]
```

Comment:
Should the Note be formatted with === ?:

And does this apply to all NOTE sections in this PR?

Reply:
I usually use the `====` myself, but I think this works ok if there is only a single following sentence but I'll change it to `====` - good spot